modules/common/route/route.go

```diff
@@ -189,3 +189,24 @@ func (r *Route) Delete(
 
 	return nil
 }
+
+// AddAnnotation - Adds annotation and merges it with the current set
+func (r *Route) AddAnnotation(anno map[string]string) {
+	r.route.Annotations = util.MergeStringMaps(r.route.Annotations, anno)
+}
+
```

Comment:
I guess we also want an `AddLabel` for `Route`?
```
// AddLabel - Adds label and merges it with the current set
func (r *Route) AddLabel(label map[string]string) {
	r.route.Labels = util.MergeStringMaps(r.route.Labels, label)
}
```

Reply:
Done!